onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb

```diff
@@ -0,0 +1,962 @@
+    "            logger.info(\"***** Eval results {} *****\".format(prefix))\n",
+    "            for key in sorted(result.keys()):\n",
+    "                logger.info(\"  %s = %s\", key, str(result[key]))\n",
+    "                writer.write(\"%s = %s\\n\" % (key, str(result[key])))\n",
+    "\n",
+    "    return results\n",
+    "\n",
+    "\n",
+    "def time_ort_model_evaluation(model_path, configs, tokenizer, prefix=\"\"):\n",
+    "    eval_start_time = time.time()\n",
+    "    result = evaluate_onnx(configs, model_path, tokenizer, prefix=prefix)\n",
+    "    eval_end_time = time.time()\n",
+    "    eval_duration_time = eval_end_time - eval_start_time\n",
+    "    print(result)\n",
+    "    print(\"Evaluate total time (seconds): {0:.1f}\".format(eval_duration_time))\n",
+    "\n",
+    "print('Evaluating ONNXRuntime full precision accuracy and performance:')\n",
+    "time_ort_model_evaluation('bert.opt.onnx', configs, tokenizer, \"onnx.opt\")\n",
+    "    \n",
+    "print('Evaluating ONNXRuntime quantization accuracy and performance:')\n",
+    "time_ort_model_evaluation('bert.opt.quant.onnx', configs, tokenizer, \"onnx.opt.quant\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3 Summary\n",
+    "In this tutorial, we demonstrated how to quantize a fine-tuned BERT model for MPRC task on GLUE data set. Let's summarize the main metrics of quantization.\n",
+    "\n",
+    "### Model Size\n",
+    "PyTorch quantizes torch.nn.Linear modules only and reduce the model from 438 MB to 181 MB. OnnxRuntime quantizes not only Linear(MatMul), but also the embedding layer. It achieves almost the ideal model size reduction with quantization.\n",
+    "\n",
+    "| Engine | Full Precision(MB) | Quantized(MB) |\n",
+    "| --- | --- | --- |\n",
+    "| PyTorch | 438 | 181 |\n",
+    "| ORT | 438 | 112 |\n",
+    "\n",
+    "### Accuracy\n",
+    "OnnxRuntime achieves a little bit better accuracy and F1 score, even though it has small model size.\n",
+    "\n",
+    "| Metrics | PyTorch Quantization | ORT Quantization |\n",
+    "| --- | --- | --- |\n",
+    "| Accuracy | 0.84559 | 0.85784 |\n",
+    "| F1 score | 0.89340 | 0.90203 |\n",
+    "| Acc and F1 | 0.86950 | 0.87994 |\n",
+    "\n",
+    "### Performance\n",
+    "\n",
+    "The evaluation data set has 408 sample. Table below shows the performance on my machine with Intel(R) Xeon(R) E5-1650 v4@3.60GHz CPU. PyTorch quantization achieves ~1.33x speedup, and ORT quantization achieves ~1.34x speedup. And ORT quantization can achieve ~1.27x speedup, comparing with PyTorch quantization. \n",
+    "You can run the [benchmark.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/benchmark.py) for comparison on more models.\n",
+    "\n",
+    "|Engine | Full Precision Latency(s) | Quantized(s) |\n",
+    "| --- | --- | --- |\n",
+    "| PyTorch | 32.9 | 24.8 |\n",
```

Comment:
I suggest append version to engine name. If you test different version, the number might be different.